idaes/models_extra/gas_distribution/unit_models/node.py

```diff
@@ -363,6 +363,31 @@ def component_mixing_rule(b, t, j):
             time, component_list, rule=component_mixing_rule
         )
 
+    def add_enthalpy_mixing_con(self):
+        """
+        Adds and equation that calculates temperature of the node from
+        temperatures, heat capacities, and flow rates of the supplies
+        and inlet pipelines.
+
+        """
+        time = self.flowsheet().time
+
+        def get_enthalpy_rate(state):
+            return state.temperature * state.flow_mol * state.cp_mol
+
+        def enthalpy_mixing_rule(b, t):
+            supplies = [self.supplies[i].state[t] for i in self.supply_set]
+            inlets = [self.inlets[i].state[t] for i in self.inlet_set]
+            return (
+                sum(get_enthalpy_rate(supply) for supply in supplies)
```

Comment:
Looking at this, is `get_enthalpy_rate()` even required? This is what the `get_enthalpy_flow_terms` method in the property package is for.

Reply:
It should not be required. I just haven't implemented `get_enthalpy_flow_terms` in this property package. Will update.